packages/dashboard/src/components/DashboardFrame/index.tsx

```diff
@@ -1,17 +1,57 @@
-import { makeStyles, useMediaQuery, Toolbar, Theme, Typography, AppBar, Grid, Box } from '@material-ui/core'
-import Color from 'color'
+import {
+    AppBar,
+    Box,
+    Grid,
+    makeStyles,
+    Toolbar,
+    Typography,
+    Drawer,
+    useMediaQuery,
+    Theme,
+    IconButton,
+} from '@material-ui/core'
+import { Menu as MenuIcon, Close as CloseIcon } from '@material-ui/icons'
+import { ErrorBoundary } from '@dimensiondev/maskbook-theme'
 import clz from 'classnames'
 import { Navigation } from './Navigation'
-import { ErrorBoundary } from '@dimensiondev/maskbook-theme'
+import { useEffect, useState } from 'react'
+import Color from 'color'
+
+export interface DashboardFrameProps extends React.PropsWithChildren<{}> {
+    title: React.ReactNode | string
+    primaryAction?: React.ReactNode
+}
 
 const useStyles = makeStyles((theme) => ({
-    root: { backgroundColor: theme.palette.background.paper },
-    temporaryDrawerPaper: {
-        backgroundColor: new Color(theme.palette.background.paper).alpha(0.8).toString(),
+    appBar: {},
+    toolbar: {
+        [theme.breakpoints.up(1184)]: {
```

Comment:
Why this magic number? We have MUI breakpoints.

Reply:
The small screen on the design is supported by 1184, and I asked the designer if we could set this magic number as our breakpoint if needed

Reply:
@xiazhiming

Reply:
yeah, use larger breakpoints